prelude/prelude.purs

```diff
@@ -98,14 +109,42 @@ module Prelude
   infixr 0 $
   infixl 0 #
 
+  -- | Applies a function to its argument
+  -- |
+  -- | ```purescript
```

Comment:
Like this:

    ```purescript
    length $ groupBy productCategory $ filter isInStock products
    ```
   
    is equivalent to
    
    ```purescript
    length (groupBy productCategory (filter isInStock (products)))
    ```

Also, you need to regenerate the docs.

Thanks.